src/scene/export.ts

```diff
@@ -14,39 +14,34 @@ export const exportToCanvas = (
   appState: AppState,
   {
     exportBackground,
-    exportPadding = 10,
+    exportPadding = DEFAULT_EXPORT_PADDING,
     viewBackgroundColor,
-    scale = 1,
   }: {
     exportBackground: boolean;
     exportPadding?: number;
-    scale?: number;
     viewBackgroundColor: string;
   },
   createCanvas: (
     width: number,
     height: number,
   ) => { canvas: HTMLCanvasElement; scale: number } = (width, height) => {
-    const tempCanvas = document.createElement("canvas");
-    tempCanvas.width = width * scale;
-    tempCanvas.height = height * scale;
-    return { canvas: tempCanvas, scale };
+    const canvas = document.createElement("canvas");
+    canvas.width = width * appState.exportScale;
+    canvas.height = height * appState.exportScale;
+    return { canvas, scale: appState.exportScale };
   },
 ) => {
   const [minX, minY, width, height] = getCanvasSize(elements, exportPadding);
 
-  const { canvas: tempCanvas, scale: newScale = scale } = createCanvas(
-    width,
-    height,
-  );
+  const { canvas, scale = 1 } = createCanvas(width, height);
```

Comment:
I've reintroduced a hardcoded `1` default scale to align with production. A default value is technically unnecessary here, but in case host apps (npm) don't use TypeScript, it's now harder to realize you're not supplying `scale` in `appState.exportScale` when calling this API programmatically.

Reply:
Sounds sensible. I keep forgetting that some people aren't using type script.